### PR TITLE
Extra test for attribute divisor reset to 0 in angle-instanced-arrays.html

### PR DESCRIPTION
**Pull request by shrekshao**
Related to workaround for latest Mac Intel opengl driver bug http://crbug.com/1144207

Test description: we want to make sure webgl does instanced draw correctly when the non-zero divisor of an attribute is reset to zero. The attribute with the new divisor == 0 should be treated as a regular vertex attribute.

The test turns divisor of color attribute to zero. with `drawArraysInstancedANGLE(gl.TRIANGLES, 3, 3, instanceCount);`, it's the last three yellow vec4 that would be accessed by vertices. The test should draw 4 yellow triangles.

Tested locally on Mac AMD (passes chrome/firefox, patched chrome), Mac Intel (fails chrome/firefox, passes patched chrome)

### CODE REVIEW

File: sdk/tests/conformance/extensions/angle-instanced-arrays.html [hunk 1]
```diff
@@ -277,6 +280,17 @@
     wtu.checkCanvasRect(gl, Math.ceil(0.25*canvas.width), 0, w, h, [0, 0, 255, 255]);
     wtu.checkCanvasRect(gl, Math.ceil(0.75*canvas.width), 0, w, h, [255, 255, 0, 255]);
 
+    debug("");
+    debug("Testing drawArraysInstancedANGLE with attributes 'divisor' reset to 0");
+    ext.vertexAttribDivisorANGLE(colorLoc, 0);
```
Comment (kenrussell) [inline sdk/tests/conformance/extensions/angle-instanced-arrays.html:285]: Could you please add another debug() call saying what the correct output is, and what the incorrect rendering looked like when this driver bug was occurring?

**Review by kenrussell — APPROVED** [review]
Great job Shrek working around this fundamental driver bug. New test section looks good; one small request. Please feel free to merge yourself afterward.

**Review by kenrussell — COMMENTED** [review]
One more thought - could you add similar code to conformance2/rendering/instanced-arrays.html ?

**Comment by shrekshao** [comment]
Updated similiar section for webgl2 instanced-arrays.html

File: sdk/tests/conformance2/rendering/instanced-arrays.html [hunk 2]
```diff
@@ -174,6 +177,45 @@
     gl.drawArraysInstanced(desktopGL['POLYGON'], 0, 6, instanceCount);
     wtu.glErrorShouldBe(gl, gl.INVALID_ENUM, "drawArraysInstanced with POLYGON should return INVALID_ENUM");
 
+    debug("Testing drawArraysInstancedANGLE with param 'first' > 0");
```
Comment (kenrussell) [inline sdk/tests/conformance2/rendering/instanced-arrays.html:180]: drawArraysInstancedANGLE -> drawArraysInstanced (here and throughout)

**Review by kenrussell — APPROVED** [review]
Thanks for focusing so well on this problem and the test Shrek - great work. One tiny comment.

**Comment by kenrussell** [comment]
By the way - if you can think of a refactoring into a separate .js file that would allow these two tests to share more code, that would be great.

**Comment by shrekshao** [comment]
I think we can defer merge into one js to a later pull request

**Pull request merged**